src/javaRosa.js

```diff
@@ -1725,76 +1680,54 @@ define([
             // hide non-itext messages unless present
             control.label.visibility = "visible_if_present";
             control.hintLabel.visibility = "visible_if_present";
-            // virtual property used to define a widget
+
             control.labelItext = {
-                visibility: 'labelItextID',
+                visibility: 'visible',
                 presence: 'optional',
+                lstring: "Label",
                 widget: function (mug, options) {
                     return itextLabelBlock(mug, $.extend(options, {
                         itextType: "label",
                         getItextByMug: function (mug) {
-                            return mug.p.labelItextID;
+                            return mug.p.labelItext;
                         },
                         displayName: "Label"
                     }));
                 },
-                lstring: "Label"
+                validationFunc: spec.control.label.validationFunc
             };
-
+            // virtual property used to define a widget
             control.labelItextID = {
-                visibility: 'visible',
+                visibility: 'labelItext',
                 presence: 'optional',
                 lstring: "Question Itext ID",
-                widget: function (mug, options) {
-                    return iTextIDWidget(mug, $.extend(options, {
-                        displayName: "Add Other Content",
-                        itextType: "label",
-                        getItextByMug: function (mug) {
-                            return mug.p.labelItextID;
-                        },
-                        forms: ['long', 'short'],
-                        isCustomAllowed: true
```

Comment:
These were dead options. None of them were used by `iTextIDWidget` (the same applies below). The history of this code is ... tangled.